packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -159,13 +160,17 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
   );
 
   return (
-    <ControlBar layout="dockedBottom">
-      {microphoneButton}
-      {cameraButton}
-      {screenShareButton}
-      {participantButton}
-      {optionsButton}
-      {endCallButton}
-    </ControlBar>
+    <Stack horizontalAlign="center">
+      <Stack.Item>
+        <ControlBar layout="horizontal">
```

Comment:
Note: this is updated to `horiztonal` from `dockedBottom` so the CallArrangement page correctly allocates space for the video gallery based on the height of this component. This is because `dockedBottom` has `position; absolute` and so it is floating and thus the `grow` property on the video gallery would not take its height into consideration for its sizing.

Reply:
Great justification. Please add this as comment in the code.

These kinds of "why" get lost very easily.

Reply:
Updated